client/my-sites/post-type-filter/index.jsx

```diff
@@ -240,6 +245,7 @@ export default flow(
 
 			const props = {
 				siteId,
+				state,
```

Comment:
Let's avoid sharing the whole state also here.

Reply:
fixed

Reply:
@millerf did you miss this one? I see it's been fixed [in the other component](https://github.com/Automattic/wp-calypso/blob/3c9e4f57c0edae442d89a5f50ceb1960a9492fe3/client/my-sites/pages/main.jsx#L173) but not [here](https://github.com/Automattic/wp-calypso/blob/3c9e4f57c0edae442d89a5f50ceb1960a9492fe3/client/my-sites/post-type-filter/index.jsx#L248).

Reply:
Yes, I overlooked it. I just pushed a fix.